src/os_maild/os_maild_client.c

```diff
@@ -672,3 +724,112 @@ MailMsg *OS_RecvMailQ_JSON(file_queue *fileq, MailConfig *Mail, MailMsg **msg_sm
 
     return NULL;
 }
+
+
+/**
+ * @brief Read cJSON and save in printed with email format
+ * @param item Pointer to the cJSON to read
+ * @param printed Body email
+ * @param body_size Maximun body message size
+ * @param tab Determine the number of tabs on each line
+ * @param counter Count the number of times that is tabulated in a line
+ */
+void PrintTable(cJSON *item, char *printed, size_t body_size, char *tab, int counter)
+{
+    char *val2;
+    int log_size;
```

Comment:
Define `log_size` as `size_t` to avoid these errors:

```
os_maild/os_maild_client.c:808:31: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
                 if (body_size > log_size) {
```